tests/flytekit/integration/remote/test_remote.py

```diff
@@ -483,3 +484,34 @@ def test_execute_workflow_with_maptask(register):
         wait=True,
     )
     assert execution.outputs["o0"] == [4, 5, 6]
+
+@pytest.mark.serial
+@pytest.mark.parametrize("gigabytes", [2, 3])
+def test_can_register_with_large_file(gigabytes):
+    with tempfile.TemporaryDirectory(dir=MODULE_PATH) as tempdir:
```

Comment:
@pingsutw I see failures in the CI - maybe we can just test with a single 3Gb file?

```suggestion
def test_can_register_with_large_file():
    gigabytes = 3
    with tempfile.TemporaryDirectory(dir=MODULE_PATH) as tempdir:
```